portfolio/src/main/webapp/script.js

```diff
@@ -98,10 +98,173 @@ function setInfoEvents() {
 function setUp() {
     setTabEvents();
     setInfoEvents();
+    getComments();
+    // setFormSubmit();
+
+    // var postTarget = document.getElementById("submitTarget");
+    // postTarget.onload = onLoadPost;
 }
 
 const tabSelectedClass = "tabSelected";
 const tabClass = "tab";
 const tabContentClass = "tabContent";
-const galleryTitle = "galleryTab";
+const SPECIAL_TAB = "special";
 const groupHeader = "groupHeader";
+
+// Servlet functions
+
+/**
+ * Fetches a message from the server and adds it to the DOM
+ */
+function getComments() {
+  const comments = fetch(`/list-comments?vis=${getVis()}`);
+  comments.then(response => response.json()).then((comments) => { 
+            handleGivenComments(comments)
+          });
+}
+
+/**The maximum number of comments that should be displayed */
+const MAX_COMMENTS = 3;
+
+/**
+ * Handles the comments from the server by converting them to text and giving them to addNameToDom()
+ */
+function handleGivenComments(comments) {
+  for (var i = 0; i < comments.length; i++) {
+    addCommentToDom(comments[i]);
+  }
+}
+
+const COMMENTS_DISPLAY = "commentsDisplay";
+
+/**
+ * Adds the comment to the page
+ */
+function addCommentToDom(comment) {
+  const commentDisplay = document.getElementById(COMMENTS_DISPLAY);
+  const obj = makeCommentElement
+    (comment, comment.commentText, comment.commentAuthor);
+
+  // add the new objects to the comment Display
+  commentDisplay.appendChild(obj);
+}
+
+/**
+ * @return HTML div object containing the comment with relevant information and styling
```

Comment:
nit: Instead of "HTML div object", you can say "HTML div element" to be more specific about what type of object it is.